sympy/printing/tests/test_tensorflow.py

```diff
@@ -12,14 +12,22 @@
 from sympy.matrices import Matrix, MatrixBase, eye, randMatrix
 from sympy.matrices.expressions import \
     Determinant, HadamardProduct, Inverse, MatrixSymbol, Trace
-from sympy.printing.tensorflow import tensorflow_code
 from sympy.tensor.array.expressions.conv_matrix_to_array import convert_matrix_to_array
 from sympy.utilities.lambdify import lambdify
 from sympy.testing.pytest import skip
 from sympy.testing.pytest import XFAIL
+from sympy.utilities.exceptions import ignore_warnings
 
 
-tf = tensorflow = import_module("tensorflow")
+with ignore_warnings(DeprecationWarning):
+    # Warnings when importing tensorflow:
+    #
+    # DeprecationWarning: The distutils package is deprecated and slated for
+    # removal in Python 3.12. Use setuptools or check PEP 632 for potential
+    # alternatives
+    from sympy.printing.tensorflow import tensorflow_code
+    tf = tensorflow = import_module("tensorflow")
```

Comment:
Similar to Aesara. When tensorflow stops using distutils these kinds of changes can be reverted.